core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -1003,4 +1015,4 @@ void testDateFormat(DateFormatTester dft) {
   private ExprValue eval(Expression expression) {
     return expression.valueOf(env);
   }
-}
+}
```

Comment:
I think you can revert all changes in this file

Reply:
Agreed. Reverted all changes.